### PR TITLE
Use Guava Strings class instead of assertj Strings class

### LINKED ISSUE #1076: Use Guava Strings class instead of assertj Strings class.
In the `ElasticJobLiteAutoConfiguration` class, import `org.assertj.core.util.Strings` class for String null or empty.

I suggest import `com.google.common.base.Strings` instead of `org.assertj.core.util.Strings`.

### PR DESCRIPTION
**Pull request by zhaoyuguang**
Fixes #1076

### CODE REVIEW

File: elasticjob-lite/elasticjob-lite-spring-boot-starter/src/main/java/org/apache/shardingsphere/elasticjob/lite/boot/ElasticJobLiteAutoConfiguration.java [hunk 1]
```diff
@@ -50,9 +50,9 @@
 @EnableConfigurationProperties(ElasticJobProperties.class)
 @Setter
 public class ElasticJobLiteAutoConfiguration implements ApplicationContextAware {
-    
+
```
Comment (dongzl) [inline elasticjob-lite/elasticjob-lite-spring-boot-starter/src/main/java/org/apache/shardingsphere/elasticjob/lite/boot/ElasticJobLiteAutoConfiguration.java]: Please keep the white space.

File: elasticjob-lite/elasticjob-lite-spring-boot-starter/src/main/java/org/apache/shardingsphere/elasticjob/lite/boot/ElasticJobLiteAutoConfiguration.java [hunk 2]
```diff
@@ -50,9 +50,9 @@
...
     private ApplicationContext applicationContext;
-    
+
```
Comment (dongzl) [inline elasticjob-lite/elasticjob-lite-spring-boot-starter/src/main/java/org/apache/shardingsphere/elasticjob/lite/boot/ElasticJobLiteAutoConfiguration.java]: Please keep the white space.

File: elasticjob-lite/elasticjob-lite-spring-boot-starter/src/main/java/org/apache/shardingsphere/elasticjob/lite/boot/ElasticJobLiteAutoConfiguration.java [hunk 3]
```diff
@@ -68,7 +68,7 @@ public void createJobBootstrapBeans() throws ClassNotFoundException {
         registerClassedJobs(elasticJobProperties, beanFactory, registryCenter, tracingConfiguration);
         registerTypedJobs(elasticJobProperties, beanFactory, registryCenter, tracingConfiguration);
     }
-
```
Comment (dongzl) [inline elasticjob-lite/elasticjob-lite-spring-boot-starter/src/main/java/org/apache/shardingsphere/elasticjob/lite/boot/ElasticJobLiteAutoConfiguration.java]: The same to above.

File: elasticjob-lite/elasticjob-lite-spring-boot-starter/src/main/java/org/apache/shardingsphere/elasticjob/lite/boot/ElasticJobLiteAutoConfiguration.java [hunk 4]
```diff
@@ -81,7 +81,7 @@ private TracingConfiguration getTracingConfiguration() {
                 "More than one [org.apache.shardingsphere.elasticjob.tracing.api.TracingConfiguration] beans found. "
                         + "Consider disabling [org.apache.shardingsphere.elasticjob.tracing.boot.ElasticjobTracingAutoConfiguration].");
     }
-    
+
```
Comment (dongzl) [inline elasticjob-lite/elasticjob-lite-spring-boot-starter/src/main/java/org/apache/shardingsphere/elasticjob/lite/boot/ElasticJobLiteAutoConfiguration.java]: The same to above.

File: elasticjob-lite/elasticjob-lite-spring-boot-starter/src/main/java/org/apache/shardingsphere/elasticjob/lite/boot/ElasticJobLiteAutoConfiguration.java [hunk 5]
```diff
@@ -97,8 +97,8 @@ private void registerClassedJobs(final ElasticJobProperties elasticJobProperties
             }
         }
     }
-    
-    private void registerTypedJobs(final ElasticJobProperties elasticJobProperties, final SingletonBeanRegistry beanFactory, 
+
```
Comment (dongzl) [inline elasticjob-lite/elasticjob-lite-spring-boot-starter/src/main/java/org/apache/shardingsphere/elasticjob/lite/boot/ElasticJobLiteAutoConfiguration.java]: The same to above.

**Review by dongzl — CHANGES_REQUESTED** [review]

File: elasticjob-lite/elasticjob-lite-spring-boot-starter/src/main/java/org/apache/shardingsphere/elasticjob/lite/boot/ElasticJobLiteAutoConfiguration.java (hunk 5 above)
Comment (zhaoyuguang) [reply elasticjob-lite/elasticjob-lite-spring-boot-starter/src/main/java/org/apache/shardingsphere/elasticjob/lite/boot/ElasticJobLiteAutoConfiguration.java]: I had fixed

**Review by terrymanu — APPROVED** [review]

**Pull request merged**